tests/conftest.py

```diff
@@ -2,6 +2,8 @@
 
 import numpy as np
 import pytest
+import tensorflow as tf
+from tensorflow._api.v1.compat import v1 as tf1
```

Comment:
This seems strange is there not an import ... version -- also import error is needed